iptables/iptables.go

```diff
@@ -173,7 +175,7 @@ func (ipt *IPTables) runWithOutput(args []string, stdout io.Writer) error {
 	args = append([]string{ipt.path}, args...)
 	if ipt.hasWait {
```

Comment:
@jonboulle It is forced in the test to reach the internal flock.
See https://github.com/coreos/go-iptables/blob/master/iptables/iptables_test.go#L82-L88.
I think a minor change to test would be better.